### ISSUE TITLE
Unable to access web GUI interface.  

### ISSUE THREAD

**Issue by DC072012** [jgyates/genmon#789]
Use the template below if you have an issue or want to report a bug. If you have a question or a feature request you can ignore the questions below. This is a template so feel free to remove any of this text. Try to include the info below when applicable.

***NOTE:*** If you are having issues with your serial connection, please read this page before posting:

https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting

If you are having other issues, please see the following page:

https://github.com/jgyates/genmon/wiki/3.5---General-Troubleshooting

You can search past issues by removing the "is:open" keywords from the search window and adding any keywords that are relevant to your issue.

If you need to send you logs and registers to the developer, if your email is setup and working properly you can click send your logs on the About page in the web interface.

### Expected Behavior

Should be able to access genmon via web interface at http://192.168.57.170:7500

### Actual Behavior

Safari can't connect to server.  I searched and searched the issues but can't figure out what to check.

### Steps to Reproduce (including precondition)

Genmon was working well.  I intended to install the Pushover system today.  Before I did that I wanted to get a backup image of the SD card.  
Apparently that was a mistake.

I shut down the Pi 3, brought the card in.  Used Apple Pi Baker on the Mac to clone the card to my hard drive.  Apple Pi Baker said it worked but I'm not sure.  Properly ejected the card, put it back in the Pi.

Genmon starts up, I get email messages saying it is active on 
IP 192.168.57.170.  If I turn my switch off on the controller genmon sends an email notifying me.  Turn switch back on I get email notification.

Somehow cloning the card must have messed something up with the web settings - I have no idea.

Can SSH into the Pi (that is how I sent the logs using [ClientInterface.py].  
Nothing changed with my router - it is pre-assigning the proper IP address.  

Can ping 192.168.57.170, median ping time 11ms.

I cannot figure out how to fix the web interface access problem since I don't know what caused it to get messed up.

Thanks in advance for any help.
### Screenshot or Pictures relating to the problem (if possible)

generator: monitor

Monitor : 

    Generator Monitor Stats : 
        Monitor Health : OK
        Controller : Nexus, Air Cooled
        Run time : Generator Monitor running for 0:30:31.
        Generator Monitor Version : V1.18.15
        Update Available : No

    Communication Stats : 
        Packet Count : M: 57266, S: 57265
        CRC Errors : 0 
        CRC Percent Errors : 0.00%
        Timeout Errors : 0
        Timeout Percent Errors : 0.00%
        Modbus Exceptions : 0
        Validation Errors : 0
        Sync Errors : 0
        Invalid Data : 0
        Discarded Bytes : 0
        Comm Restarts : 0
        Packets Per Second : 62.54
        Average Transaction Time : 0.0307 sec

    Platform Stats : 
        CPU Temperature : 110.30 F
        Pi Model : Raspberry Pi 3 Model B Plus Rev 1.3
        Pi CPU Frequency Throttling : OK
        Pi ARM Frequency Cap : OK
        Pi Undervoltage : OK
        CPU Utilization : 3.0%
        OS Name : Raspbian GNU/Linux
        OS Version : 9 (stretch)
        System Uptime : 0:31:12
        Network Interface Used : wlan0
        WLAN Signal Level : -59 dBm
        WLAN Signal Quality : 51/70
        WLAN Signal Noise : -256 dBm
        WLAN ESSID : djnet-1
        System Time : Sunday October 16, 2022 13:45:28

### Your Environment

- Generator Model: 005882-0
- Genmon Logs: Submit via the About page (sent using manual interface)
- Genmon Version: current

**Comment by DC072012** [comment]
Found another issue thread, thought this might add info.  Still have the problem however:

Ran:
pi@genpi [~] cd genmon
pi@genpi [~/genmon] ./startgenmon.sh restart

Restarting genmon python scripts
/usr/local/lib/python3.5/dist-packages/OpenSSL/crypto.py:14: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import utils, x509
Stopping....
Stopping genserv.py
Stopping genmon.py
Starting....
Starting /home/pi/genmon/genmon.py 
Starting /home/pi/genmon/genserv.py 

Then ran (some of these output messages seem old):
cat /var/log/genserv.log

Output:
2019-09-05 19:41:55,328 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-09-05 20:38:43,393 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-09-05 21:21:03,058 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-09-06 08:14:22,007 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-09-06 08:17:40,517 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-10-07 14:02:39,477 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-10-07 14:06:27,463 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-10-10 15:47:48,042 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-10-12 10:54:00,028 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-10-27 16:12:08,906 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2019-10-27 16:12:16,392 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-10-27 16:12:41,191 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2019-10-27 16:12:48,577 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-11-10 12:17:22,369 : Warning: Reading Config File (value not in list): controllertype : 
2019-11-10 14:45:23,678 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2019-11-10 14:45:27,880 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2019-11-10 14:45:38,253 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-11-10 14:45:38,378 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2019-11-10 14:45:53,274 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2019-11-10 14:46:02,268 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-11-10 19:03:36,499 : Warning: Reading Config File (value not in list): controllertype : 
2019-11-15 12:33:08,540 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2019-11-15 12:33:10,187 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2019-11-15 12:33:17,500 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2019-11-16 11:56:04,886 : Warning: Reading Config File (value not in list): controllertype : 
2019-11-16 11:56:12,826 : Warning: Reading Config File (value not in list): controllertype : 
2019-11-16 11:56:58,415 : Warning: Reading Config File (value not in list): controllertype : 
2019-11-19 14:22:29,166 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2019-11-19 14:22:30,896 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2019-11-19 14:22:38,696 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-01-16 18:54:05,398 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-01-16 18:54:07,478 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2020-01-16 18:54:14,844 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-01-16 21:58:55,721 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2020-01-16 21:59:03,125 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-01-17 15:41:40,975 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-01-17 16:05:02,155 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-01-17 16:05:06,440 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2020-01-17 16:05:15,197 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-01-20 11:01:39,523 : Warning: Reading Config File (value not in list): controllertype : 
2020-01-22 11:12:23,896 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-02-16 15:37:22,604 : Warning: Reading Config File (value not in list): controllertype : 
2020-02-21 13:52:50,178 : Warning: Reading Config File (value not in list): controllertype : 
2020-02-26 12:40:18,883 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-02-26 12:40:23,137 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2020-02-26 12:44:59,600 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-03-15 17:02:31,640 : Warning: Reading Config File (value not in list): controllertype : 
2020-04-29 19:16:04,226 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-04-29 19:16:06,337 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2020-04-29 19:16:14,212 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-05-02 08:51:30,787 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-05-02 12:36:52,923 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-05-05 13:41:40,318 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-05-07 13:47:34,734 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-05-12 10:58:17,665 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-05-19 15:41:22,592 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-05-19 15:41:49,438 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2020-05-19 15:41:56,931 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-05-20 16:26:13,626 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-05-20 16:26:18,227 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2020-05-20 16:26:39,038 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-05-20 20:03:06,745 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-05-20 20:03:11,072 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart /etc/genmon/
2020-05-20 20:03:19,401 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-06-11 06:59:07,134 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-06-11 06:59:09,187 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-06-11 06:59:51,222 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-06-17 06:09:51,824 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-06-17 06:09:54,048 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-06-17 06:10:02,139 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-06-17 06:10:02,187 : Error in app.run: [Errno 98] Address already in use : genserv.py:1866
2020-06-17 06:10:04,190 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-06-17 08:04:56,312 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-06-17 08:28:09,658 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -b -c /etc/genmon/
2020-06-29 17:25:31,229 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-06-29 17:25:35,747 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-06-29 17:25:44,625 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-07-06 22:32:10,906 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-07-06 22:32:12,804 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-07-06 22:32:20,484 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-07-09 12:36:25,160 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-07-12 07:05:26,550 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-07-12 07:05:30,927 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-07-12 07:05:43,053 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-07-25 07:24:23,609 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-07-25 07:24:25,303 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-07-25 07:24:33,804 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-07-25 07:24:53,310 : Warning: Reading Config File (value not in list): controllertype : 
2020-07-25 07:25:17,290 : Warning: Reading Config File (value not in list): controllertype : 
2020-07-25 07:25:27,058 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-07-25 07:25:37,159 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-07-27 07:39:00,086 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-08-01 21:58:15,712 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-08-01 21:58:40,028 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-08-01 21:58:48,423 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-08-05 11:35:59,169 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-08-05 11:36:01,176 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-08-05 11:36:09,025 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-08-22 22:03:25,350 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-08-22 22:03:27,792 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-08-22 22:03:46,956 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-09-14 10:35:50,602 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-09-14 10:35:54,204 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-09-14 10:36:02,594 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-09-14 10:36:12,585 : Root
2020-09-15 09:17:05,022 : Root
2020-09-15 09:17:13,103 : Root
2020-09-15 14:20:23,344 : Root
2020-09-16 18:02:16,051 : Root
2020-09-17 08:32:06,893 : Root
2020-09-17 09:15:32,170 : Root
2020-09-18 12:18:51,283 : Root
2020-09-18 19:13:48,196 : Root
2020-09-18 20:25:09,696 : Root
2020-09-20 07:15:41,169 : Root
2020-09-20 08:58:13,733 : Root
2020-09-20 08:58:38,071 : Root
2020-09-20 08:58:40,110 : Root
2020-09-20 10:15:34,149 : Warning: Reading Config File (value not in list): controllertype : 
2020-09-20 10:16:39,744 : Root
2020-09-20 10:16:39,751 : Root
2020-09-20 13:00:38,147 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-09-20 13:00:39,980 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-09-20 13:01:03,613 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-09-24 17:52:12,687 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -b -c /etc/genmon/
2020-09-29 08:30:49,439 : Warning: Reading Config File (value not in list): controllertype : 
2020-09-29 08:31:07,631 : Warning: Reading Config File (value not in list): controllertype : 
2020-10-03 16:22:23,390 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-10-07 08:30:35,759 : Warning: Reading Config File (value not in list): controllertype : 
2020-10-08 17:23:11,992 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-10-12 06:33:55,387 : Warning: Reading Config File (value not in list): controllertype : 
2020-10-15 13:01:23,338 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-10-16 17:08:52,879 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-10-19 13:33:44,082 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-10-24 17:45:55,531 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-10-24 17:46:03,173 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-10-24 17:46:11,618 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-10-26 21:53:09,752 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-10-26 21:53:11,548 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-10-26 21:53:19,988 : Starting /home/pi/genmon/genserv.py, Port:8000, Secure HTTP: False, SelfSignedCert: True
2020-10-31 20:15:16,983 : Warning: Reading Config File (value not in list): controllertype : 
2020-11-02 12:55:25,308 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-11-02 12:55:33,750 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2020-11-02 14:10:11,071 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-11-02 14:10:12,812 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-11-02 14:10:20,398 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2020-11-05 20:23:14,845 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-11-05 20:23:16,671 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-11-05 20:23:24,389 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2020-11-05 22:36:01,473 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-11-05 22:36:03,261 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-11-05 22:36:11,044 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2020-11-06 15:56:26,858 : Warning: Reading Config File (value not in list): controllertype : 
2020-11-06 16:01:26,530 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-11-06 16:01:34,951 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2020-11-06 16:02:19,076 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-11-06 16:02:27,547 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2020-11-06 16:21:07,470 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-11-06 16:21:15,074 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2020-11-10 06:31:12,079 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-11-10 06:31:15,691 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-11-10 06:31:23,574 : Error setting up 2FA: 'NoneType' object has no attribute 'SenderAccount' : genserv.py:1932
2020-11-10 06:31:23,575 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2020-11-13 21:52:38,279 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -b -c /etc/genmon/
2020-11-20 16:57:45,731 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-11-20 16:57:49,171 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-11-20 16:59:13,200 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2020-12-13 13:05:43,542 : Warning: Reading Config File (value not in list): controllertype : 
2020-12-17 19:23:39,486 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2020-12-17 19:23:41,481 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2020-12-17 19:23:49,413 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-01-02 17:02:22,818 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-01-02 17:02:24,651 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-01-02 17:02:32,543 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-01-13 11:50:18,975 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-01-14 06:52:14,798 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-01-14 06:52:22,803 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-01-14 06:53:57,374 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-01-14 06:54:05,478 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-01-16 08:42:36,558 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-01-16 08:42:41,170 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-01-16 08:42:55,670 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-01-16 08:42:56,198 : Error in app.run: [Errno 98] Address already in use : genserv.py:2088
2021-01-16 08:42:56,198 : Retrying app.run()
2021-01-16 08:42:58,202 : Error in app.run (2): [Errno 98] Address already in use : genserv.py:2099
2021-01-16 08:48:16,432 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-01-16 12:38:22,381 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-01-16 12:41:59,533 : Warning: Reading Config File (value not in list): controllertype : 
2021-01-16 12:44:08,476 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-01-16 12:44:12,666 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-01-16 12:44:33,176 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-01-20 14:25:58,816 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-01-20 20:03:39,856 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -b -c /etc/genmon/
2021-01-22 17:19:30,450 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-01-22 17:19:34,560 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-01-22 17:19:42,490 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-01-28 06:48:20,079 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -b -c /etc/genmon/
2021-02-01 09:25:02,521 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-02-01 09:25:05,114 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-02-01 09:25:13,108 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-02-02 10:01:40,919 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-02-02 10:01:48,854 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-02-04 16:39:57,790 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-02-04 16:39:59,499 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-02-04 16:40:07,419 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-02-04 18:41:24,601 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-02-04 18:41:32,560 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-02-04 18:43:48,765 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-02-04 18:43:56,749 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-02-04 18:48:30,010 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -b -c /etc/genmon/
2021-02-06 07:01:02,028 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-02-06 07:01:03,770 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-02-06 07:01:11,776 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-02-06 11:38:32,027 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-02-06 11:38:40,015 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-02-10 11:06:23,657 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-02-10 11:06:25,486 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-02-10 11:06:33,467 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-02-28 10:22:55,367 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-02-28 10:22:57,484 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-02-28 10:23:05,478 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-03-02 12:47:19,651 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-03-02 12:47:21,351 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-03-02 12:47:29,617 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-03-02 20:21:07,175 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-03-02 20:21:08,935 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-03-02 20:21:17,100 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-03-03 10:50:23,097 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-03-03 10:50:31,291 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-03-04 11:33:35,756 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-03-04 11:33:37,710 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-03-04 11:33:45,993 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-03-16 17:52:41,006 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -b -c /etc/genmon/
2021-03-17 17:38:19,764 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-03-17 17:38:21,589 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-03-17 17:38:29,700 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-04-07 14:20:49,948 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-04-07 14:20:51,858 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-04-07 14:20:59,991 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-05-10 16:14:43,715 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-05-10 16:14:45,552 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-05-10 16:14:53,666 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-05-20 15:50:04,014 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-05-20 15:50:06,214 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-05-20 15:50:14,665 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-05-22 14:44:48,095 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-05-22 14:44:50,304 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-05-22 14:44:58,709 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-05-22 14:45:54,920 : Warning: Reading Config File (value not in list): controllertype : 
2021-05-22 14:46:03,358 : Warning: Reading Config File (value not in list): controllertype : 
2021-05-29 15:10:39,992 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-05-29 15:10:41,854 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-05-29 15:10:50,001 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-05-29 15:11:42,523 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-05-29 15:11:44,128 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-05-29 15:11:52,279 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-06-12 13:04:47,762 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-06-12 13:04:49,619 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-06-12 13:04:57,891 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-06-20 07:54:15,739 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-06-20 07:54:17,495 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-06-20 07:54:25,660 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-07-06 16:05:31,282 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-07-06 16:05:33,294 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-07-06 16:05:41,637 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-07-10 14:21:46,861 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-07-10 14:21:48,901 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-07-10 14:21:57,180 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-07-31 12:50:25,231 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-07-31 12:50:27,235 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-07-31 12:50:35,486 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-08-01 17:03:20,890 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-08-01 17:03:22,670 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-08-01 17:03:30,854 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-08-29 18:38:07,133 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-08-29 18:38:09,273 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-08-29 18:38:17,591 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-10-11 17:41:32,159 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-10-11 17:41:34,127 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-10-11 17:41:42,315 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-10-16 17:55:22,307 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-10-16 17:55:24,116 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-10-16 17:55:32,261 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-11-17 08:19:14,932 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n
2021-11-17 08:19:17,182 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -c /etc/genmon/
2021-11-17 08:24:38,757 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-11-17 16:01:28,299 : Warning: Reading Config File (value not in list): controllertype : 
2021-11-24 12:33:10,496 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2021-11-24 12:33:12,474 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2021-11-24 12:33:22,161 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-12-06 07:10:06,124 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2021-12-06 07:10:08,097 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2021-12-06 07:10:17,881 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2021-12-07 07:47:20,012 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2021-12-07 07:47:21,800 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2021-12-07 07:47:31,544 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-03-04 19:40:10,870 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2022-03-04 19:40:13,693 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-03-04 19:40:49,797 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-03-12 07:47:04,871 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2022-03-12 07:47:06,825 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-03-12 07:47:16,625 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-04-05 08:48:29,247 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2022-04-05 08:48:31,225 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-04-05 08:48:41,008 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-04-19 13:25:58,203 : Warning: Reading Config File (value not in list): controllertype : 
2022-04-19 13:27:22,910 : Warning: Reading Config File (value not in list): controllertype : 
2022-04-19 13:27:35,420 : Warning: Reading Config File (value not in list): controllertype : 
2022-04-20 21:07:43,572 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2022-04-20 21:07:45,491 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-04-20 21:07:56,611 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-05-28 16:44:48,010 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2022-05-28 16:44:50,034 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-05-28 16:45:00,896 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-06-05 19:06:56,091 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2022-06-05 19:06:57,978 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-06-05 19:07:07,653 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-06-21 22:48:14,865 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2022-06-21 22:48:16,768 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-06-21 22:48:26,845 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-07-26 07:09:42,760 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2022-07-26 07:09:45,044 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-07-26 07:09:54,868 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-07-26 07:11:16,524 : Warning: Reading Config File (value not in list): controllertype : 
2022-07-26 07:11:37,060 : Warning: Reading Config File (value not in list): controllertype : 
2022-08-22 09:28:21,730 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2022-08-22 09:28:23,999 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-08-22 09:28:59,318 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-08-23 08:38:31,870 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-08-23 08:38:41,908 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-08-26 07:11:00,917 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-08-26 07:11:10,673 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-09-19 08:18:20,266 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2022-09-19 08:18:22,402 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-09-19 08:18:32,150 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-09-19 21:47:27,573 : Script: /bin/bash /home/pi/genmon/genmonmaint.sh -u -n -p 3
2022-09-19 21:47:29,443 : Script: /bin/bash /home/pi/genmon/startgenmon.sh restart -p 3 -c /etc/genmon/
2022-09-19 21:47:39,269 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-10-16 13:15:01,940 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
2022-10-16 16:30:06,186 : Starting /home/pi/genmon/genserv.py, Port:7500, Secure HTTP: False, SelfSignedCert: True
pi@genpi [~/genmon] cat /var/log/genloader.log
2019-03-05 01:07:35,513 : Warning: Missing entry: genexercise , adding entry
2019-03-05 01:07:35,563 : Copying genexercise.conf to /etc/
2019-05-01 19:12:20,763 : Warning: Missing entry: genemail2sms , adding entry
2019-05-01 19:12:20,821 : Copying genemail2sms.conf to /etc/
2019-06-21 18:11:34,762 : Warning: Missing entry: gentankutil , adding entry
2019-06-21 18:11:34,807 : Copying gentankutil.conf to /etc/genmon/
2019-08-03 17:25:46,224 : Warning: Missing entry: genalexa , adding entry
2019-08-03 17:25:47,034 : Copying genalexa.conf to /etc/genmon/
2020-02-26 12:40:23,791 : Warning: required library pysnmp not installed. Attempting to install....
2020-02-26 12:44:52,813 : Warning: Missing entry: gensnmp , adding entry
2020-02-26 12:44:52,865 : Copying gensnmp.conf to /etc/genmon/
2020-06-11 06:59:09,825 : Warning: required library ldap3 not installed. Attempting to install....
2020-06-17 06:10:05,090 : Warning: Missing entry: genmon
2020-06-17 06:10:05,090 : Warning: Missing entry: genserv
2020-06-17 06:10:05,090 : Warning: Missing entry: gengpio
2020-06-17 06:10:05,091 : Warning: Missing entry: gengpioin
2020-06-17 06:10:05,091 : Warning: Missing entry: genlog
2020-06-17 06:10:05,091 : Warning: Missing entry: gensms
2020-06-17 06:10:05,092 : Warning: Missing entry: gensms_modem
2020-06-17 06:10:05,092 : Warning: Missing entry: genpushover
2020-06-17 06:10:05,092 : Warning: Missing entry: gensyslog
2020-06-17 06:10:05,093 : Warning: Missing entry: genmqtt
2020-06-17 06:10:05,093 : Warning: Missing entry: genslack , adding entry
2020-06-17 06:10:12,258 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,261 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,691 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,698 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,705 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,712 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,739 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,739 : Warning: Missing entry: genslack
2020-06-17 06:10:12,739 : Warning: Missing entry: genexercise , adding entry
2020-06-17 06:10:12,740 : Error in WriteSection: Section already exist.
2020-06-17 06:10:12,748 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,755 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,762 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,769 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,776 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,783 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,783 : Warning: Missing entry: genexercise
2020-06-17 06:10:12,784 : Warning: Missing entry: genemail2sms , adding entry
2020-06-17 06:10:12,784 : Error in WriteSection: Section already exist.
2020-06-17 06:10:12,791 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,798 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,805 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,811 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,818 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,825 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,826 : Warning: Missing entry: genemail2sms
2020-06-17 06:10:12,826 : Warning: Missing entry: gentankutil , adding entry
2020-06-17 06:10:12,827 : Error in WriteSection: Section already exist.
2020-06-17 06:10:12,833 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,840 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,847 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,854 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,861 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,868 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,868 : Warning: Missing entry: gentankutil
2020-06-17 06:10:12,868 : Warning: Missing entry: genalexa , adding entry
2020-06-17 06:10:12,869 : Error in WriteSection: Section already exist.
2020-06-17 06:10:12,875 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,883 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,890 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,897 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,903 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,910 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,911 : Warning: Missing entry: genalexa
2020-06-17 06:10:12,911 : Warning: Missing entry: gensnmp , adding entry
2020-06-17 06:10:12,911 : Error in WriteSection: Section already exist.
2020-06-17 06:10:12,918 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,925 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,932 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,939 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,946 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,953 : Error in WriteValue: File contains parsing errors: /etc/genmon/genloader.conf
	[line  3]: 'ion name with the following parameters\n'
2020-06-17 06:10:12,953 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,954 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,954 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,955 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,955 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,955 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,956 : Error in MyConfig:ReadValue: postloaddelay: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,956 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,957 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,957 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,957 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,958 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,958 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,959 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,959 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,959 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,960 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,960 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,961 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,961 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,961 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,962 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,962 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,963 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,963 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,963 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,964 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,964 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,965 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,965 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,965 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,966 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,966 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,967 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,967 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,967 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,968 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,968 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,968 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,969 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,969 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,970 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,970 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,970 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,971 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,971 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,972 : Error in MyConfig:ReadValue: enable: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,972 : Error in MyConfig:ReadValue: hardstop: 'list' object has no attribute 'lower' : myconfig.py:86
2020-06-17 06:10:12,973 : Error in MyConfig:ReadValue: priority: int() argument must be a string or a number, not 'list' : myconfig.py:90
2020-06-17 06:10:12,973 : Error validating config for gensnmp : 'list' object has no attribute 'split' : genloader.py:255
2020-06-17 06:10:12,974 : Error validating config. Exiting
2020-07-06 22:32:13,659 : Warning: Missing entry: gentemp , adding entry
2020-07-06 22:32:13,729 : Copying gentemp.conf to /etc/genmon/
2020-08-22 22:03:28,671 : Warning: required library smbus not installed. Attempting to install....
2020-08-22 22:03:38,196 : Warning: Missing entry: gentankdiy , adding entry
2020-08-22 22:03:38,740 : Copying gentankdiy.conf to /etc/genmon/
2020-09-20 13:00:40,858 : Warning: required library pyotp not installed. Attempting to install....
2020-11-10 06:31:16,611 : Warning: Missing entry: gengpioledblink , adding entry
2020-11-10 06:31:16,804 : Copying gengpioledblink.conf to /etc/genmon/
2020-11-20 16:57:50,041 : Warning: required library psutil not installed. Attempting to install....
2021-01-16 12:38:16,710 : Warning: unable to find config file: /etc/genmon/genloader.conf Copying file to /etc/genmon/ directory.
2021-01-16 12:38:16,720 : Warning: Missing entry: gengpioledblink , adding entry
2021-06-12 13:04:50,689 : Updated entry gengpio.conf
2021-06-12 13:04:50,698 : Copying gengpio.conf to /etc/genmon/
2021-11-17 08:19:17,863 : Warning: required library flask not installed. Attempting to install....
2021-11-17 08:19:17,868 : Error in CheckBaseSoftware: [Errno 2] No such file or directory: 'pip3' : genloader.py:184
2021-11-17 08:19:17,869 : Installing python3-pip
2021-11-17 08:20:51,214 : Warning: required library pyserial not installed. Attempting to install....
2021-11-17 08:21:09,613 : Warning: required library crcmod not installed. Attempting to install....
2021-11-17 08:21:20,714 : Warning: required library pyowm not installed. Attempting to install....
2021-11-17 08:21:42,687 : Warning: required library pytz not installed. Attempting to install....
2021-11-17 08:21:56,278 : Warning: required library pysnmp not installed. Attempting to install....
2021-11-17 08:22:21,137 : Warning: required library ldap3 not installed. Attempting to install....
2021-11-17 08:22:44,068 : Warning: required library smbus not installed. Attempting to install....
2021-11-17 08:22:55,403 : Warning: required library pyotp not installed. Attempting to install....
2021-11-17 08:23:06,414 : Warning: required library psutil not installed. Attempting to install....
2021-11-17 08:23:21,743 : Warning: required library chump not installed. Attempting to install....
2021-11-17 08:23:32,980 : Warning: required library twilio not installed. Attempting to install....
2021-11-17 08:23:54,622 : Warning: required library paho-mqtt not installed. Attempting to install....
2021-11-17 08:24:06,271 : Warning: required library pyopenssl not installed. Attempting to install....
2022-03-04 19:40:15,632 : Warning: required library spidev not installed. Attempting to install....
2022-03-04 19:40:22,637 : Error in CheckBaseSoftware  : b'/usr/local/lib/python3.5/dist-packages/OpenSSL/crypto.py:14: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.\n  from cryptography import utils, x509\n'
2022-03-04 19:40:34,595 : Error in InstallLibrary using pip : spidev : UnInstall: False: b'/usr/local/lib/python3.5/dist-packages/OpenSSL/crypto.py:14: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.\n  from cryptography import utils, x509\n'
2022-03-04 19:40:34,606 : Warning: Missing entry: gencthat , adding entry
2022-03-04 19:40:34,674 : Warning: Missing entry: gencthat
2022-03-04 19:40:34,675 : Adding entry: genloader
2022-03-04 19:40:34,694 : Running one time maintenance check
2022-03-04 19:40:41,926 : Error in GetLibararyVersion using pip : pyowm: b'/usr/local/lib/python3.5/dist-packages/OpenSSL/crypto.py:14: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.\n  from cryptography import utils, x509\n'
2022-03-04 19:40:41,936 : Copying gencthat.conf to /etc/genmon/
2022-08-22 09:28:38,175 : Installing cmake...
2022-08-22 09:28:49,907 : Error in ExecuteCommandList  : b'debconf: unable to initialize frontend: Dialog\ndebconf: (Dialog frontend will not work on a dumb terminal, an emacs shell buffer, or without a controlling terminal.)\ndebconf: falling back to frontend: Readline\ndebconf: unable to initialize frontend: Readline\ndebconf: (This frontend requires a controlling tty.)\ndebconf: falling back to frontend: Teletype\ndpkg-preconfigure: unable to re-open stdin: \n'
2022-08-22 09:28:51,302 : Warning: Missing entry: genmopeka , adding entry
2022-08-22 09:28:51,374 : Warning: Missing entry: genmopeka
2022-08-22 09:28:51,394 : Copying genmopeka.conf to /etc/genmon/
pi@genpi [~/genmon]

**Comment by jgyates** [comment]
I received your logs. Nothing outstadning there. I would not assume that just because nothing has changed in your router that that is not the issue. The fact that your pi rebooted may be triggering some protection / blocking in your router. Do you have an ASUS router. They have some features in some models that have "learning" security that I have seen block web servers on your local network.


Try to ping the pi with the network name you have assigned (from the logs abvove)

    ping genpi

or 

     ping genpi.local

If either of these work then subustitue the named value (e.g. genpi.local) in the URL 

      http://genpi.local:7500

If that does not work then I would start looking at your router settings.

Also, do you have remote access to your pi setup via your router? If you do try disabling it.

The logs are showing that everything is working as expected.

You could also try rebooting your pi.

**Comment by jgyates** [comment]
Also, you can search past issues with the "Router" tag and see a few issues similar to what you are describing.

**Comment by DC072012** [comment]
OK, thanks for the info so far.  Another data point.

If I reboot the Pi, the web interface will not work.  If I do this:
 cd genmon
  ./startgenmon.sh restart

the web interface works immediately after the restart.  Not sure the router would cause this based on reading other posts.  What else might cause this?  thanks again.

**Comment by jgyates** [comment]
Try this:

1) reboot your pi
2) log in via ssh
3) issue this command:

        curl -I  http://192.168.57.170:7500/

what is the output of the above command? 

4) try to access genmon with ClientInterface.py. Is this successful?

5) now do this:

      cd genmon
     ./startgenmon.sh restart

Then repeat steps 3 and 4

Also I noticed this issue from the past?

https://github.com/jgyates/genmon/issues/271

Generally when troubleshooting an issue like this try disabling any external access  or port forwarding at the router and try to access the pi. Since you can access the pi on your network, just not the web server, and the server appears to be functioning

**Comment by DC072012** [comment]
Thanks for the help.  I re-read #271.  It looks like that was a problem caused by trying to access genmon via remote access through the router.  After that problem I took your advice and set up a VPN on Digital Ocean where I can access items on my local network without port forwarding the devices.  

I ran the following tests as you outlined with no port forwarding turned on at all at the router.  Rebooting the Pi results in no connection, restarting genmon after that fixes the problem.  Seems like the reboot isn't starting something right away???  

Thanks again.

Linux genpi 4.14.98-v7+ #1200 SMP Tue Feb 12 20:27:48 GMT 2019 armv7l

  _______ _______ ______      _______ ___ 
 |   _   |   _   |   _  \    |   _   |   |
 |.  |___|.  1___|.  |   |   |.  1   |.  |
 |.  |   |.  __)_|.  |   |   |.  ____|.  |
 |:  1   |:  1   |:  |   |   |:  |   |:  |
 |::.. . |::.. . |::.|   |   |::.|   |::.|
 `-------`-------`--- ---'   `---'   `---'
                                          

Last login: Sun Oct 16 21:16:26 2022 from 192.168.57.93
pi@genpi [~] sudo reboot
Connection to 192.168.57.170 closed by remote host.
Connection to 192.168.57.170 closed.
 ~ % ssh pi@192.168.57.170
pi@192.168.57.170's password: 
Linux genpi 4.14.98-v7+ #1200 SMP Tue Feb 12 20:27:48 GMT 2019 armv7l

  _______ _______ ______      _______ ___ 
 |   _   |   _   |   _  \    |   _   |   |
 |.  |___|.  1___|.  |   |   |.  1   |.  |
 |.  |   |.  __)_|.  |   |   |.  ____|.  |
 |:  1   |:  1   |:  |   |   |:  |   |:  |
 |::.. . |::.. . |::.|   |   |::.|   |::.|
 `-------`-------`--- ---'   `---'   `---'
                                          

Last login: Sun Oct 16 21:28:41 2022
pi@genpi [~] curl -I  http://192.168.57.170:7500/
curl: (7) Failed to connect to 192.168.57.170 port 7500: Connection refused
pi@genpi [~] python3 ClientInterface.py -a 192.168.57.170
python3: can't open file 'ClientInterface.py': [Errno 2] No such file or directory
pi@genpi [~] cd genmon
pi@genpi [~/genmon] ./startgenmon.sh restart
Restarting genmon python scripts
/usr/local/lib/python3.5/dist-packages/OpenSSL/crypto.py:14: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
  from cryptography import utils, x509
Stopping....
Stopping genserv.py
Stopping genmon.py
Starting....
Starting /home/pi/genmon/genmon.py 
Starting /home/pi/genmon/genserv.py 

pi@genpi [~/genmon] curl -I  http://192.168.57.170:7500/
HTTP/1.0 200 OK
Content-Length: 2245
Content-Type: text/html; charset=utf-8
Last-Modified: Tue, 20 Sep 2022 01:47:29 GMT
Cache-Control: no-cache, no-store, must-revalidate, public, max-age=0
Expires: 0
ETag: "1663638449.1661212-2245-3473673333"
Date: Mon, 17 Oct 2022 01:29:56 GMT
Pragma: no-cache
Server: Werkzeug/1.0.1 Python/3.5.3

pi@genpi [~/genmon] python3 ClientInterface.py -a 192.168.57.170
OK : Auto, Off - Ready
>exit
pi@genpi [~/genmon]

**Comment by jgyates** [comment]
One thing to keep in mind. Once your system boots, genmon will wait 30 seconds before starting. Try it again but wait about 30 seconds after the initial login via ssh.

You can check the script that starts genmon like this:

      sudo crontab -e

you may be promoted to chose an editor, if you are press the number that uses the nano editor

This will bring up a script file and it should have the following line in it:

     @reboot sleep 30 && /bin/bash /home/pi/genmon/startgenmon.sh start

If it does not then add this line, but only if it is not there already. Starting it twice would be problematic.

if you do change the contents of the cron job then you can exit by typing Ctrl+X. If you changed the file it will ask you if you want to save it. Press Y then Enter to save

**Comment by DC072012** [comment]
This line was there:
@reboot sleep 30 && /bin/bash /home/pi/genmon/startgenmon.sh start

I changed it to 60 seconds:
@reboot sleep 60 && /bin/bash /home/pi/genmon/startgenmon.sh start

I rebooted the Pi 2 times and both times the web interface became available without needing to restart genmon.  

Not sure why the 30 seconds isn't enough.  Are there any Pi system updates I should run?  I'm not sure the system has been updated since I bought it.

thanks again.

**Comment by jgyates** [comment]
I run this once a week:

     sudo apt-get update
     sudo apt-get dist-upgrade

This makes sure that security updates and bug fixes are applied to the operating system. Once it is updated it is a good idea to reboot. There are some other utilities that you can run that will tell if you a reboot is required but rebooting if you have not run the update in a long time is probably needed,
